### DISCUSSION TITLE
Documentation (questions/suggestion?)

### DISCUSSION THREAD

**Discussion by gsal** [reflex-dev/reflex#130] [General]
The Components Overview section contains a "Pages" article that, in my opinion, is out of place and redundant with the actual "Pages" section:

```
Components    | 
    Overview  |          Pages (article out of place)
    Props     | 
    Pages     | 
```
Another opinion (disclaimer: English is my second language, although I believe I know it well, by now :wink: )
```
Events Reference
    Triggers
        **on_mouse_enter** 
        **on_mouse_over**
            look identical
            I vote for on_mouse_enter
            "to be over" something does not sound like an event, more like a condition/state
            "to enter" is an event
        **on_mouse_leave**
        **on_mouse_out** 
            look identical
            I vote for on_mouse_leave
            "to be out" does not sound like an event, more like a condition/state
            "to leave" is an event
```
My 2 cents

**Comment by Alek99** [comment]
Thank you for your input and for checking out the docs.

The reason we have a pages section is because we plan on doing some more advanced stuff with it later like dynamic routing and setting titles/descriptions/image previews for individual page.  Titles/descriptions/image are done I just need to make docs for it. I agree right now it looks a little redundant because all the information is a little simple but in the future it will be good to have as a reference.


I see you have linked the event triggers section as https://pynecone.io/docs/events-reference/triggers is this the page you mean to be redundant? On that page I just wanted to show every one some examples of common event triggers but I agree some of them will probably be not used very often. We could potentially simplify event triggers not include them all but I wanted to give as much flexibility as possible to the users

**Comment by gsal** [comment]
Yes, in that events-reference/triggers page is where I find the duplicate mouse events I speak of.

**Reply by Alek99** [reply]
Oh I see now thanks for pointing this out this is something I can simplify. I go most of them from here https://reactjs.org/docs/events.html.

 I agree with you I'll look into taking out `on_mouse_out` and `on_mouse_over`. Seem redundant

**Comment by FHU-yezi** [comment]
Does our documentation opersource? Maybe be can contribute to it.

**Reply by Alek99** [reply]
Not yet we talked about it but were a little worried about potentially leaking a database secret or something in there. Would could Open-source the docs in the future though
